### ISSUE TITLE
Anyone work with D3.js?

### ISSUE THREAD

**Issue by mkchandler** [codeforokc/discussions#10]
I'm wanting to make [D3.js](http://d3js.org/) a topic for [CodeAcross](http://www.codeforamerica.org/events/codeacross-2015/) next month. Is anyone here knowledgable about D3.js, or know someone who is that would like to do a short presentation and intro to D3?

D3 is something that will be very useful for visualizing some of the open data that we have access to. Something I would like to tackle during CodeAcross is creating a visualization of city spending.

**Comment by joekarl** [comment]
@jvrousseau or @DevinClark and probably an open call to the okcjs twitter account will get ahold of who you want.

**Comment by jagthedrummer** [comment]
I've done a bit with D3 at a couple of different levels.

NVD3 makes it super easy to drop in some pre-built charts and I've used it on a number of projects. http://nvd3.org/

I've also done a little bit with raw D3, which has kind of... unusual requirements for structuring your code.

**Comment by mkchandler** [comment]
I remember your presentation on NVD3, that looks like good stuff. I'd like to see if we could use a [treemap](http://bl.ocks.org/mbostock/4063582) to visualize the spending or budget. I've seen it done for some other cities, though I'm having trouble finding links at this moment.

**Comment by mkchandler** [comment]
Here's one for Philly: http://budget.brettmandel.com/

**Comment by jagthedrummer** [comment]
Oh, nice. Do we have that kind of tree like info?

**Comment by mkchandler** [comment]
I'm not sure haha. But we can get it if don't already have it.

**Comment by jvrousseau** [comment]
Another good d3 piece for visualizing a budget is force layout, a la: http://www.nytimes.com/interactive/2012/02/13/us/politics/2013-budget-proposal-graphic.html. A great presentation on it: http://vallandingham.me/abusing_the_force.html

**Comment by mkchandler** [comment]
@jvrousseau Yeah, that's pretty awesome. Love the transitions between the different "views".

**Comment by mkchandler** [comment]
To give an update on this topic, we found [someone to give a presentation on D3.js](https://twitter.com/DarrenJaws). I also received the FY2015 budget data from the city via an [Open Records request](http://www.okc.gov/clerk/OpenRecordsRequest/Forms/SubmitOpenRecordsRequest.aspx). I will start a new repo and get it uploaded.

**Comment by mkchandler** [comment]
Repo has been created: https://github.com/codeforokc/okc-budget-data

Further discussion can take place over there.
